windows/scripts/gendriver.sh

```diff
@@ -38,9 +40,29 @@ if [ ! -d $ISO_DIR ]; then
     exit 1
 fi
 
-mkdir -p "$OUTPUT_DIR"
+mkdir -p "$OUTPUT_DIR" "$KB_DIR"
 
-./scripts/extract_kernels.py  --iso-dir "$ISO_DIR" -o "$OUTPUT_DIR"
+download_patch()
+{
+    local URL="$1"
+    local FILE="$(basename $URL)"
+    local DEST_FILE="$KB_DIR/$FILE"
+    if [ -f "$DEST_FILE" ]; then
+        echo "$DEST_FILE already exists, skipping."
+    else
+        $WGET "$DEST_FILE" "$URL"
+    fi
+}
+
+# Download patches for driver signature checks
+download_patch https://download.microsoft.com/download/C/8/7/C87AE67E-A228-48FB-8F02-B2A9A1238099/Windows6.1-KB3033929-x64.msu
+download_patch https://download.microsoft.com/download/3/7/4/37473F39-5728-4153-9A25-64C09DE9ED52/Windows6.1-KB3033929-x86.msu
+
+# Extract kernels from KBs first
+./scripts/extract_kernels.py  --iso-dir "$KB_DIR" -o "$OUTPUT_DIR"
+
+# Extract ISOs
+# ./scripts/extract_kernels.py  --iso-dir "$ISO_DIR" -o "$OUTPUT_DIR"
```

Comment:
Why is this line still here? Should it be removed?

Reply:
Forgot to uncomment after debugging